lib/archiving.js

```diff
@@ -114,6 +114,11 @@ var generateHeaders = function generateHeaders(config) {
 *   the URL for 10 minutes. To generate a new URL, call the
 *   {@link OpenTok#getArchive OpenTok.getArchive()} or
 *   {@link OpenTok#listArchives OpenTok.listArchives()} method.
+* 
+* @property {String} multiArchiveTag
+*   The tag of the archive, with which to start. This is unique for each archive within session. 
+*   <code>multiArchiveTag</code> is not case-sensitive (e.g. 'MyArchive' and 'myarchive' are 
+*   treated as same)
```

Comment:
This is not quite accurate. See the docs for `multiArchiveTag` in the REST API docs: https://tokbox.com/developer/rest/#start_archive.

Reply:
My mistake. I took from https://github.com/opentok/api-review/tree/master/archiving-api